docs/use-service.md

```diff
@@ -89,10 +89,10 @@ Note: Skadi files are split into 1° by 1° grids. File names refer to the latit
 
 If you’re building in Amazon AWS we recommend using machines in the `us-east` region (the same region as the S3 bucket) and use the following endpoints for increased performance:
 
-* `https://s3.amazonaws.com/elevation-tiles-prod/terrarium/{z}/{x}/{y}.png`
-* `https://s3.amazonaws.com/elevation-tiles-prod/normal/{z}/{x}/{y}.png`
-* `https://s3.amazonaws.com/elevation-tiles-prod/geotiff/{z}/{x}/{y}.tif`
-* `https://s3.amazonaws.com/elevation-tiles-prod/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz`
+* `https://s3.amazonaws.com/elevation-tiles-prod/v2/terrarium/{z}/{x}/{y}.png`
+* `https://s3.amazonaws.com/elevation-tiles-prod/v2/normal/{z}/{x}/{y}.png`
+* `https://s3.amazonaws.com/elevation-tiles-prod/v2/geotiff/{z}/{x}/{y}.tif`
+* `https://s3.amazonaws.com/elevation-tiles-prod/v2/skadi/{N|S}{y}/{N|S}{y}{E|W}{x}.hgt.gz`
```

Comment:
Pending Amazon discussion, we should also note v1 tiles are still available?

Reply:
These are unversioned now (without `v2/`), right? Do we tell people how to use S3 versioning to get at the v1 state?

Reply:
Fixed in b7eb8c5.

There isn't an easy way to use versioned objects from the HTTP endpoint, so my inclination would be to not tell people about it.

Reply:
Mentioning it would be useful, simply as a reminder to our future selves that it is versioned ;)

Reply:
Added a mention in 5c159d3.